src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -20,53 +20,215 @@ float CallImplicitGemmDynamic(const miopen::Handle& handle,
+
+            int h_tilda_slice = h_tilda_right - h_tilda_left;
+            int w_tilda_slice = w_tilda_right - w_tilda_left;
+
+            int num_of_gemms  = x_tilda * y_tilda;
+
+            int dtile_iy    = 0;
+            int dtile_ix    = 0;
+            int dtile_dy    = dilation_h / gcd_stride_dilation_h;
+            int dtile_dx    = dilation_w / gcd_stride_dilation_w;
+            int dtile_y     = y_tilda;
+            int dtile_x     = x_tilda;
+            int dtile_h     = h_tilda;
+            int dtile_w     = w_tilda;
+            int dslice_y    = 0;
+            int dslice_x    = 0;
+            int dslice_h    = h_tilda_slice;
+            int dslice_w    = w_tilda_slice;
+            int dslice_h_left   = h_tilda_left;
+            int dslice_w_left   = w_tilda_left;
+            int __pack0     = 0;
+// clang-format on
+#if 0
+            MIOPEN_LOG_I2("hi:" << hi << ", wi:" << wi << ", n:" << n << ", k:" << k << ", c:" << c
+                                << ", ho:"
+                                << ho
+                                << ", wo:"
+                                << wo
+                                << ", stride_h:"
+                                << stride_h
+                                << ", stride_w:"
+                                << stride_w
+                                << ", dilation_h:"
+                                << dilation_h
+                                << ", dilation_w:"
+                                << dilation_w
+                                << ", pad_h:"
+                                << pad_h
+                                << ", pad_w:"
+                                << pad_w
+                                << ", y:"
+                                << y
+                                << ", x:"
+                                << x
+                                << ", gcd_stride_dilation_h:"
+                                << gcd_stride_dilation_h
+                                << ", gcd_stride_dilation_w:"
+                                << gcd_stride_dilation_w
+                                << ", y_tilda:"
+                                << y_tilda
+                                << ", x_tilda:"
+                                << x_tilda
+                                << ", h_tilda:"
+                                << h_tilda
+                                << ", w_tilda:"
+                                << w_tilda
+                                << ", h_tilda_left:"
+                                << h_tilda_left
+                                << ", w_tilda_left:"
+                                << w_tilda_left
+                                << ", h_tilda_slice:"
+                                << h_tilda_slice
+                                << ", w_tilda_slice:"
+                                << w_tilda_slice);
+#endif
```

Comment:
Could this deprecated code section be removed?

Reply:
removed